packages/graphql-testing/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/graphql-testing",
-  "version": "7.0.1",
+  "version": "7.0.2",
```

Comment:
Changesets will handle this for you

Reply:
Thanks! I just noticed that in contribution guide and reverted :)